README.md

```diff
@@ -90,9 +90,9 @@ def mystery5(s)
 end
 ```
 
-- What is mystery5("hi")?
-- What is mystery5("")?
-- What is mystery5("Hi, there!")?
+- What is mystery5("hi")? **
+- What is mystery5("")? **blank**
```

Comment:
You should always put things that return a String in double quotes. For this answer, the answer should be "" (empty String) but saying "blank" is confusing.